src/components/tab-panel/AccessControlTabPanel.vue

```diff
@@ -52,30 +53,39 @@
 
 <script setup>
 import * as GroupService from 'src/services/GroupService';
-import GroupsTable from 'components/tables/GroupsTable.vue';
+import * as RoleService from 'src/services/RoleService';
+import AccessControlTable from 'components/tables/AccessControlTable.vue';
 import {
   onMounted,
   onUnmounted,
   ref,
+  computed,
   watch,
 } from 'vue';
 import ReloadGroupsEvent from 'src/composables/events/ReloadGroupsEvent';
+import ReloadRolesEvent from 'src/composables/events/ReloadRolesEvent';
 import DialogEvent from 'src/composables/events/DialogEvent';
 
-const groups = ref([]);
-const groupName = ref('');
+const rows = ref([]);
+const name = ref('');
 const currentPage = ref(0);
 const maxPage = ref(0);
 const elementsPerPage = ref(10);
 const totalElements = ref(0);
 const loading = ref(false);
+
 let reloadGroupsEventRef;
+let reloadRolesEventRef;
 
 const props = defineProps({
   entity: {
     type: Object,
     required: true,
   },
+  accessControlType: {
+    type: String,
+    required: true,
+  },
   type: {
```

Comment:
rename to `subAccessControlType` or another thing but not `type`

Reply:
subAccessControlType -> type
type -> subType